src/pages/ImportData.js

```diff
@@ -125,14 +125,19 @@ const ImportData = () => {
 
     await Utils.switchCase(format,
       ['PFB', async () => {
-        const { jobId } = await Ajax().Workspaces.workspace(namespace, name).importPFB(url)
+        const { jobId } = await Ajax().Workspaces.workspace(namespace, name).importJob(url, 'pfb')
         asyncImportJobStore.update(Utils.append({ targetWorkspace: { namespace, name }, jobId }))
         notifyDataImportProgress(jobId)
       }],
       ['entitiesJson', async () => {
         await Ajax().Workspaces.workspace(namespace, name).importJSON(url)
         notify('success', 'Data imported successfully.', { timeout: 3000 })
       }],
+      ['tdrexport', async () => {
```

Comment:
Do we expect more formats with a similar path? If so, maybe we should extract this into `startImport(format)`?

Reply:
I am not aware of any more formats on the horizon … but who knows, some may appear …